perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -365,7 +412,7 @@ def AnalyzeListResults(final_result, result_consistent, list_count,
                                         LIST_RESULT_SUFFIX_CONSISTENT)
   result_string_latency = '%s%s' % (list_scenario, LIST_RESULT_SUFFIX_LATENCY)
   result_string_inconsistency_window = '%s%s' % (list_scenario,
-      LIST_RESULT_SUFFIX_INCONSISTENCY_WINDOW)  # noqa
+                               LIST_RESULT_SUFFIX_INCONSISTENCY_WINDOW)  # noqa
```

Comment:
I do not think noqa is needed here, if you put list_scenario to the same line as LIST_RESULT..